### DISCUSSION TITLE
Pi-Hole db mapping in proxmox

### DISCUSSION THREAD

**Discussion by tismofied** [netalertx/NetAlertX#493] [Q&A]
I have Pi-hole running as an LXC on proxmox.  I also have Pi-alert running in a docker container inside a separate LXC.  I wanted to have Pi-Hole separate from anything else because I didn't want it to be included with other things that might break and I need Pi-hole to work all the time for internet before my family pick up the pitch forks. 
I didn't know about Pi-alert before I had installed Pi-Hole which was about a year ago.  
How can I map /etc/pihole/pihole-ftl.db >>>> pialert ?

**Comment by jokob-sk** [comment]
Hi there!

You need to either change the path to the PiHole DB location or move the PiHole DB file to `/etc/pihole/pihole-FTL.db`:

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/d0f347a0-b01f-41de-872f-d92b54728faa)

I'll make the setting editable in the next version. You can in the meantime save the settings and then edit the `config/pialert.conf` file directly.

Hope this helps,
j

**Comment by tismofied** [comment]
Hello there! 
Thank you for answering. So even though they are in completely different proxmox containers? Because I can't access pihole files from pialert container.

**Comment by jokob-sk** [comment]
I don't have experience with proxmox containers, but you can map files to docker containers from a shared filesystem so I imagine it's possible on Proxmox as well.

**Comment by itpilot** [comment]
guys please can you help the noob

firstly, Wow !! NETALERTX is a Brilliant Product
Thank you for doing this

I am using NETALERTX on a Raspberry PI 4 in Docker

my question is will it be wise to Also Run PI-HOLE on the Same Raspberry PI?

any help greatly appreciated

**Reply by jokob-sk** [reply]
Hey, yes, that should work
